Installer.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#
+# File: Installer.sh
+# Description: Starts the Jazz serverless installer wizard.
+#
+# ---------------------------------------------
+# Usage:
+# ---------------------------------------------
+# To Installer, run:
+# ./Installer.sh
+# ---------------------------------------------
+
+# Variables section
+
+# Installation directory
+INSTALL_DIR=`pwd`
+
+# Log file to record the installation logs
+LOG_FILE_NAME=installer_setup.out
+LOG_FILE=`realpath $INSTALL_DIR/$LOG_FILE_NAME`
+JAZZ_BRANCH="master"
+
+# Default verbosity of the installation
+VERBOSE=0
+
+function start_wizard {
+    # Set the permissions
+    chmod -R +x $INSTALL_DIR/installscripts/*
+    mkdir -p $INSTALL_DIR/installscripts/sshkeys/dockerkeys
+
+    # Call the python script to continue installation process
+    cd $INSTALL_DIR/installscripts/wizard
+
+    python ./run.py $JAZZ_BRANCH $INSTALL_DIR
+
+    setterm -term linux -fore green
+    setterm -term linux -fore default
+}
+
+# Running the selector
+while [ $# -gt 0 ] ; do
```

Comment:
@suryajak @bleggett Should we need to handle verbosity for the Installer.sh?

Reply:
I don't think it's worth making `Installer.sh` especially verbose, it does barely anything now except launch the python script, which has never been affected by the installer's `VERBOSE` flag anyhow.